### DISCUSSION TITLE
API for Verification of QR

### DISCUSSION THREAD

**Discussion by avinash2006** [Digitaler-Impfnachweis/certification-apis#234] [Q&A]
Hi Experts,
We are delivering webapps to our customers. one such app needs a functionality to scan the QR code and fetch details out of it.
As I see, there are functions which are for Android and iOS (with SDKs). But is there any API path which decodes and validates the QR code?
Or anyother service that aligns with Federal Law on Covid Rules?

Regards,
Avinash

**Comment by farhadnowzari** [comment]
Hi, I also would need information regarding this.
I have already found this https://github.com/Digitaler-Impfnachweis/certification-apis . But it is not clear enough.

@avinash2006 have your already found anything regarding this topic?

**Reply by thinkberg** [reply]
The certification is exactly that "certification". For verification we provide a service independent of the RKI offering.

**Reply by farhadnowzari** [reply]
Wow, ok that's cool.

Where can I find a documentation for it?

We are developing a self service which we need users to scan the QR code through our web application.

**Reply by avinash2006** [reply]
> 

Hi @farhadnowzari ,
I have not found any other solution yet.

@thinkberg: can you provide more information on the certification verification service? We are looking to integrate the covid app functionalities in our apps....

**Reply by thinkberg** [reply]
The verification is a different offering from UBIRCH. I suggest to contact support@ubirch.com regarding this.
